packages/ui-kit/src/rendering/renderBlockElement.ts

```diff
@@ -28,6 +28,18 @@ const getBlockElementRenderer = <T>(
 
     case 'linear_scale':
       return renderers.linearScale as BlockElementRenderer<T> | undefined;
+
+    case 'toggle_switch':
+      return renderers.toggleSwitch as BlockElementRenderer<T> | undefined;
+
+    case 'radio_button':
+      return renderers.radioButton as BlockElementRenderer<T> | undefined;
+
+    case 'checkbox':
+      return renderers.checkbox as BlockElementRenderer<T> | undefined;
+
+    case 'time_picker':
+      return renderers.timePicker as BlockElementRenderer<T> | undefined;
```

Comment:
I think this switch just exist for backward compatibility. There is no need for new cases.